cpu/cc26xx_cc13xx/include/cc26xx_cc13xx.h

```diff
@@ -141,7 +141,7 @@ typedef enum IRQn
 /** @ingroup cpu_specific_peripheral_memory_map
   * @{
   */
-#define FLASH_BASE         0x00000000 /**< FLASH base address */
+//#define FLASH_BASE         0x00000000 /**< FLASH base address */
```

Comment:
Why?

Reply:
That is important, this address is not being used by any board currently, however in the implementation I tried to use it but it is incorrect for the cc26x0 chip (the correct one is 0x40030000), and surely it will also be incorrect for the other boards, therefore I consider the right time to delete it to avoid making future mistakes.

Reply:
`FLASH_BASE` is the address of the flash itself (where ROM starts, 0-128K - 55). `FLASH_BASEADDR` is the one with the address you need since that is the register bank that controls the flash peripheral (which is used by `setup_trim_device` to set some initial values AFAIK). So it shouldn't be removed, instead do `s/FLASH_BASE/FLASH_BASEADDR/` on vendor code :wink:

Reply:
in the vendor file they call it `FLASHMEM_BASE 0x00000000`, I am concerned that future developers may have the same confusion when renaming. also a warning comment might help.

Reply:
`FLASH_BASE` is not used by RIOT, so no need to change the vendor file
`CPU_FLASH_BASE` is used, so you can do

```C
#define CPU_FLASH_BASE FLASHMEM_BASE
```

Reply:
currently only one file uses this variable (flash_base) so I'll change it for now.